cmd/machine-config-operator/bootstrap.go

```diff
@@ -86,6 +93,7 @@ func runBootstrapCmd(cmd *cobra.Command, args []string) {
 		Etcd:                    bootstrapOpts.etcdImage,
 		SetupEtcdEnv:            bootstrapOpts.setupEtcdEnvImage,
 		InfraImage:              bootstrapOpts.infraImage,
+		KubeClientAgentImage:    bootstrapOpts.kubeClientAgentImage,
```

Comment:
nit: the struct is already `Images` `KubeClientAgentImage` the Image suffix is redundant.

Reply:
updated